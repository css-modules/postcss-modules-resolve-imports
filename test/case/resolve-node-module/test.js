'use strict';

const setup = require('../../setup');

test('resolve-node-module', () => {
  const {resulting, exports: tokens} = setup(
    'local-by-default',
    'extract-imports',
    'scope',
    'self'
  )(__dirname);

  expect(resulting).toMatchSnapshot();
  expect(tokens).toMatchSnapshot();
});
