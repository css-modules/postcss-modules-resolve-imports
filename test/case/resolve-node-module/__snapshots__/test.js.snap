// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resolve-node-module 1`] = `
"/* imported from node_modules/button/button.css */._button_button
{
  /*common button styles*/
}/* imported from source.css */._source_continueButton
{
  color: green;
}
"
`;

exports[`resolve-node-module 2`] = `
Object {
  "continueButton": "_source_continueButton _button_button",
}
`;
